update_language_files.py

```diff
@@ -2,9 +2,17 @@
 import sys
 import io
 import datetime
+import json
 import pprint
```

Comment:
I think this pgk needs to be added to the (optional) dev dependencies in our documentation:

https://github.com/bit-team/backintime/blob/dev/CONTRIBUTING.md#build--install